src/west/configuration.py

```diff
@@ -233,7 +233,7 @@ def _location(cfg: ConfigFile, topdir: Optional[PathType] = None) -> str:
         if 'BSD' in plat:
             return '/etc/westconfig'
 
-        if 'CYGWIN' in plat:
+        if 'CYGWIN' in plat or 'MSYS_NT' in plat:
```

Comment:
nothing wrong on this line, but why is there no `signed-off-by` entry in the commit message ?